### DISCUSSION TITLE
Password changed, cannot enter anymore

### DISCUSSION THREAD

**Discussion by pwvandeursen** [alexjustesen/speedtest-tracker#346] [Q&A] [answered]
I changed the loginname and password, and apparently did something wrong. Can't log in anymore.

I deleted the packed and re-installed from scratch, but still not able to enter with the admin@example.com password combo. Any idea how to reset?

Working in docker in a synology system.

thanks
Partrick

**Comment by mikeydevelops** [comment]
How did you change the password? Did you edit the database entry directly? Did you use the UI? If you used the UI --  I changed mine today with the new release. I just had to delete cookies and it worked fine.

If you edited the database entry directly, the password should be hashed using bcrypt with at least 10 cost.

**Reply by pwvandeursen** [reply]
Through the UI, will try to reset it through the UI,

**Comment by alexdelprete** [comment]
You need to delete the database when reinstalling from scratch, data is saved in that. If you want to preserve existing data, you need to edit the password directly in the database.

**Reply by mikeydevelops** [reply]
No! Don't delete the database, are you trolling?

**Reply by alexdelprete** [reply]
> No! Don't delete the database, are you trolling?

No, you're not reading: I said "from scratch". And I added that it's possible to edit the pw in the db directly.

**Reply by mikeydevelops** [reply]
You said, from scratch in your edit. I was commenting on your first reply. You can see edit history.

![image](https://user-images.githubusercontent.com/10907429/213921237-3d04e88e-bfc8-4094-a6e1-5ca178ddfb5c.png)

**Reply by alexdelprete** [reply]
Right, I specified for trolls like you. Then I reedited to add the db edit.

**Reply by pwvandeursen** [reply]
so stupid question, how do I delete the database?

**Reply by alexdelprete** [reply]
> so stupid question, how do I delete the database?

If you use sqlite, it's in the volume mapped folder. If you use mariadb/mysql, you need to use a tool to delete the db.

Show me your docker-compose file, in the volumes: section there's the folder with the sqlite db.

**Reply by pwvandeursen** [reply]
Success. Removed the database.sqlite file in the docker folder, and re-installed.

thanks!

**Reply by alexdelprete** [reply]
don't thank the troll. ;)

please close the issue if it's ok now.

**Comment by alexjustesen** [comment, marked as answer]
https://github.com/alexjustesen/speedtest-tracker/pull/719 incoming in the next release which ads `app:reset-user-password` as a command.
